### PR TITLE
Add PR guide

### PR DESCRIPTION
**Pull request by lewispb**
Closes https://github.com/BiggerPockets/patterns/issues/40

### CODE REVIEW

File: _patterns/creating-a-pr.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+categories: Practices
+name: Pull Requests
+---
+
+What makes a good pull request?
+-------------------------
+
+* Write a clear, concise PR title *and* description which explain the change being made to the system in the PR
+* Include a link to the ticket/story from the pull request
+* If there's anything your unsure of or would like the reviewer to focus on please mention that in the description
```
Comment (SpiderMasterMind) [inline _patterns/creating-a-pr.md:11]:
```suggestion
* If there's anything you're unsure of, or would like the reviewer to focus on please mention that in the description
```

File: _patterns/creating-a-pr.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* Before marking the PR as ready for review, take a minute to do the first round of code review yourself. If there's anything obvious to you, address those issues
+
+Assigning reviewers
+-------------------------
+
+Pull Reminders will randomly assign a reviwer. Also consider requesting additional review. Factors that you might consider whilst selecting additional reviewers:
+
+- The original author of the code being changed may have valuable insight
+- A specific developer may have expertise in the subject of the PR (e.g. email system, search, Redis, Marketplace)
+- Your PR may be interesting to others if it introudces a new concept or framework
```
Comment (SpiderMasterMind) [inline _patterns/creating-a-pr.md:22]:
```suggestion
- Your PR may be interesting to others if it introduces a new concept or framework
```

File: _patterns/creating-a-pr.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+What makes a good pull request?
+-------------------------
+
+* Write a clear, concise PR title *and* description which explain the change being made to the system in the PR
+* Include a link to the ticket/story from the pull request
+* If there's anything you're unsure of, or would like the reviewer to focus on please mention that in the description
+* Include some simple guidelines on what QA might be needed
+* Before marking the PR as ready for review, take a minute to do the first round of code review yourself. If there's anything obvious to you, address those issues
```
Comment (EddM) [inline _patterns/creating-a-pr.md:13]: suggest adding something along the lines of this, to get people to fire up their work in a review app real quick to make sure everything is in place, and they haven't missed some seed data or something:

```suggestion
* Before marking the PR as ready for review, take a minute to do the first round of code review yourself. If there's anything obvious to you, address those issues
* If your work requires manual acceptance testing, do a cursory smoke test on your review app to ensure everything is in place (data, API keys, etc.) for QA Engineers to perform verification
```

Comment (lewispb) [reply _patterns/creating-a-pr.md:13]: Good idea!

**Pull request merged**